### PR TITLE
Upgrade benchmark.fyi to Rails 6.0

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @evanphx @nateberkopec, 

This PR upgrades the project to Rails 6.0. I hope that it helps to 

Some of the changes that were required:

- Correct parent class for migration file
- Install webpacker
- Stop using JavaScript (I don't see why we were loading application.js anyway)
- Change the way that we send the body in the controller tests
- Test that everything works in development and test environments

Please check it out and let me know if you have any questions/recommendations. 😄 

Thanks!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,47 +1,41 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.3.0"
+ruby '2.7.1'
```
Comment (nateberkopec) [inline Gemfile:4]: Travis can't run if the gemfile says 2.7.1 but the travis.yml says 2.4

Comment (etagwerker) [reply Gemfile:4]: good point! I updated Travis's configuration 👍

**Comment by etagwerker** [comment]
@nateberkopec ping 😃

**Comment by etagwerker** [comment]
Closing this because `benchmark-ips` now uses [this project](https://github.com/fastruby/benchmark.fyi) to share benchmarks: https://github.com/evanphx/benchmark-ips/blob/master/lib/benchmark/ips/share.rb#L10

**Pull request closed without merging**